docs/src/modules/spring/pages/access-control.adoc

```diff
@@ -21,7 +21,7 @@ Here is an example ACL on a method:
 
 [source, java]
 ----
-include::example$spring-acl-examples/src/main/java/com/example/MyAction.java[tag=allow-deny]
+include::example$spring-doc-snippets/src/main/java/com/example/acl/MyAction.java[tag=allow-deny]
```

Comment:
moved other snippets to `spring-doc-snippets`

Reply:
not in diff, so pasting here. 

```
[source, java, indent=0]
```

We haven't used that trick consistently, but we should. That ensures that the snippet doesn't have any indentation.

There are few more underneath. I know this was already like that, but since we are touching this page, we can fix this as well.

Reply:
added